assets/constants.js

```diff
@@ -18,5 +18,214 @@ var constants = {
     "Statistics", "Trigonometry", "Welcome"],
   "DISABLED_EXPLORATION_IDS": ["5"],
   "TESTING_CONSTANT": "test",
-  "LIBRARY_TILE_WIDTH_PX": 208
+  "LIBRARY_TILE_WIDTH_PX": 208,
+
+  "//": "Whether to allow custom event reporting to Google Analytics.",
+  "CAN_SEND_ANALYTICS_EVENTS": false,
+
+  "//": "The type of the response returned by a handler when an exception is raised.",
```

Comment:
I'm not sure why we're moving these to constants.json. Aren't they only used in the backend?

(Note that this file should only be for constants shared between the frontend and the backend.)